index.html

```diff
@@ -131,21 +141,25 @@ <h3 class="subheading">Some more context on the tech piece...</h3>
         through something called "platform cooperatives," where workers and
         users own the tech companies they make possible. I took a class through
         The New School and Mondgragon University in 2020 called "Platform
-        Cooperatives Now." This led me into a volunteer project seeking to
-        develop a co-op food delivery app. I was more involved in coordinating
+        Cooperatives Now." 
+        </br> 
+        </br>  
+        This led me into <strong> a volunteer project seeking to develop a co-op food delivery app. </strong> I was more involved in coordinating
         the volunteer software developers among us, but it gave me a taste for
         the grounded quality of building code and the role coding can play in a
         modern economy. I since applied and was accepted into Code the Dream in
         early 2022, where I am learning JavaScript, HTML, CSS, and related
         introductory skills, which will be followed by a course in either React.js or Node.js.
       </p>
```

Comment:
Wie haben Ihnen Ihre früheren Erfahrungen im Freiwilligenprojekt mit der Coop Food Delivery App beim Lernen mit React.js oder Node.js geholfen? /How did your former experience in the volunteer project with coop food delivery app help you with learning with React.js or Node.js?